src/platform/plugins/shared/discover/public/application/main/components/top_nav/use_esql_variables.ts

```diff
@@ -53,7 +53,7 @@ export const useESQLVariables = ({
   onUpdateESQLQuery: (query: string) => void;
 }): {
   onSaveControl: (controlState: Record<string, unknown>, updatedQuery: string) => Promise<void>;
-  getActivePanels: () => ControlPanelsState<ESQLControlState> | undefined;
+  getActivePanels: () => ControlPanelsState | undefined;
```

Comment:
should we re-add the generic bit here too?

```suggestion
  getActivePanels: () => ControlPanelsState<ESQLControlState> | undefined;
```